Code/Editor/ViewPane.cpp

```diff
@@ -87,29 +87,15 @@ class ViewportTitleExpanderWatcher
                                 }
 
                                 // Handle labels with submenus
-                                if (auto toolLabel = qobject_cast<QLabel*>(toolWidget))
+                                if (auto toolLabel = qobject_cast<QToolButton*>(toolWidget))
                                 {
                                     if (!toolLabel->isVisible())
                                     {
                                         // Manually turn the custom context menus into submenus
-                                        if (toolLabel->objectName() == "m_fovStaticCtrl")
+                                        if (toolLabel->menu() != nullptr)
```

Comment:
nit: This could just be `if (toolLabel->menu())` (I think generally preferred but as-is is totally acceptable too)

Reply:
Will fix :)